### ISSUE TITLE
vmware_guest.py implement cluster support

### ISSUE THREAD

**Issue by ikkaro** [ansible/ansible-modules-extras#2912]
##### ISSUE TYPE
- Feature Idea
##### COMPONENT NAME

vmware_guest.py
##### ANSIBLE VERSION

```
ansible 2.2.0
```
##### CONFIGURATION

Default configuration
##### OS / ENVIRONMENT

N/A
##### SUMMARY

This module should support clusters in a vsphere environment.
Should be excellent to replace(or add a new one) the variable esxi_hostname by cluster_name or select one of them and automatically select one host to deploy de vm.
Keep in mind that are pleople automating in single servers or in a complete cluster.
##### STEPS TO REPRODUCE

N/A
##### EXPECTED RESULTS

N/A
##### ACTUAL RESULTS

```
N/A
```

**Comment by ansibot** [comment]
@chrrrles ping, this issue is waiting for your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@jctanner, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by jctanner** [comment]
@ikkaro I added a slight enhancement for cluster support out of necessity. Would you mind testing and giving feedback?

https://github.com/ansible/ansible-modules-extras/pull/3243

needs_info

**Comment by ansibot** [comment]
@ikkaro ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ikkaro ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by jctanner** [comment]
I'm 99% certain the latest code supports clusters. If not, please open a new issue in ansible/ansible or migrate this one.
